src/scenes/QtumPrediction/components/TopActions/index.js

```diff
@@ -13,53 +13,56 @@ import {
   MenuItem,
   Card,
   withStyles,
+  withWidth,
 } from '@material-ui/core';
 
 import { SortBy } from 'constants';
 import styles from './styles';
 
 @injectIntl
 @withStyles(styles, { withTheme: true })
+@withWidth()
 @inject('store')
 @observer
 export default class TopActions extends Component {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     noCreateEventButton: PropTypes.bool,
+    width: PropTypes.string.isRequired,
   };
 
   static defaultProps = {
     noCreateEventButton: false,
   };
 
   render() {
-    const { classes, noCreateEventButton, fontSize, store, store: { qtumPrediction } } = this.props;
+    const { classes, noCreateEventButton, fontSize, store, store: { qtumPrediction }, width } = this.props;
     const { createEvent } = store;
 
     return (
       <Grid container className={classes.dashboardActionsWrapper}>
-        <Grid item xs={8}>
+        <Grid item xs={6}>
           {!noCreateEventButton && (
             <Button
               variant="raised"
-              size="medium"
+              size={width === 'xs' ? 'small' : 'medium'}
               color="primary"
               className={classes.createEventButton}
               onClick={createEvent.open}
             >
-              <AddIcon fontSize={fontSize} />
+              <AddIcon className={classes.createEventButtonIcon} fontSize={fontSize} />
               <FormattedMessage id="create.dialogTitle" defaultMessage="CREATE AN EVENT" />
```

Comment:
This text is missing on XS size:
![image](https://user-images.githubusercontent.com/4350404/45741230-49a08780-bc21-11e8-89dd-a53059038c6f.png)

Reply:
It's also missing the `Sort By:` text.

Reply:
![image](https://user-images.githubusercontent.com/18548885/45741753-f92e2780-bc2a-11e8-873f-354145686725.png)

Reply:
Why is it different from what I see?

Reply:
was seeing an old cached version sorry.